src/react/components/Button.js

```diff
@@ -2,9 +2,12 @@ import React from 'react';
 import './Button.css';
 
 function Button({ text, onClick, mode, disableProp }) {
-  const css =
+  if (mode === 'leer') {
+    return null;
+  }
```

Comment:
Ich verstehe nicht, was 'leer' sein und machen soll? 🤔

Reply:
ah, jetzt versteh ich das, wenn `mode === 'noBtn'` dann soll der Button gar nicht angezeigt werden? Das würde ich nicht so kompliziert machen, sondern einfach in der jeweiligen Komponente die diesen Button benutzt mit einer Bedingung den Button anzeigen oder eben nicht. Wenn ich das richtig sehe dann einfach in der `popup` Komponente. Ein Button in dem drinen geschaut wird, ob es ihn überhaupt geben soll ist etwas verwirrend :)

Reply:
ist jetzt in popup :)